src/prototype_room_my.js

```diff
@@ -346,6 +346,35 @@ Room.prototype.executeRoom = function() {
     }
   }
 
+  let carryHelpInterval = config.nextRoom.ticksUntilHelpCheck;
+  if (Game.time % carryHelpInterval) {
+    this.memory.Energy += this.energyAvailable;
```

Comment:
`this.memory.Energy` => `this.memory.energyAvailableSum` or something like that.